server/src/test/java/io/crate/expression/scalar/ExtractFunctionsTest.java

```diff
@@ -106,7 +106,7 @@ public void testExtractWeek() throws Exception {
     public void testExtractDay() throws Exception {
         assertEvaluate("extract(day from timestamp_tz)", 15);
         assertEvaluate("extract(day_of_month from timestamp_tz)", 15);
-        assertEvaluate("extract(day from INTERVAL '14 years 58 months 1250 days 49 hours' DAY TO HOUR)", 6);
+        assertEvaluate("extract(day from INTERVAL '14 years 58 months 1250 days 49 hours' DAY TO HOUR)", 1252);
```

Comment:
This was wrong before, so, I don't know if we should consider this PR as a fix instead, wdyt?